rootfs/ingest/python/qserv/version.py

```diff
@@ -41,5 +41,5 @@
 # ---------------------------------
 _LOG = logging.getLogger(__name__)
 
-REPL_SERVICE_VERSION = 22
+REPL_SERVICE_VERSION = 24
```

Comment:
For the record. The  version number in the `main` branch of Qserv is 25. Version 24 should still work.